src/psyclone/tests/domain/common/transformations/invoke_trans_test.py

```diff
@@ -0,0 +1,429 @@
+# -----------------------------------------------------------------------------
```

Comment:
Please fix pylint unidiomatic-checks issues in this file.

Reply:
All checks modified to use `isinstance()` which removes pylint warnings.